.qlty/hooks/pre-commit.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Install with:
+#
+# ln -s ../../.qlty/hooks/pre-commit.sh .git/hooks/pre-commit
+# chmod +x .git/hooks/pre-commit
```

Comment:
A `qlty git-hooks` command would be useful to auto symlink this stuff for you -- as well as a `qlty git-hooks init` subcommand to setup a basic default. Just a thought.

Reply:
Agreed, will work on that next